classes/Http/Controller/Reviewer/TalksController.php

```diff
@@ -113,11 +114,15 @@ public function rateAction(Request $request)
                 'rating' => 'required|integer',
             ]);
 
-            return $this->talkHandler
+            $content = (string) $this->talkHandler
                 ->grabTalk((int) $request->get('id'))
                 ->rate((int) $request->get('rating'));
+
+            return $content;
         } catch (ValidationException $e) {
-            return false;
+            $content = '';
         }
+
+        return new Response($content);
```

Comment:
In a separate PR, we should probably not just return a plain text response with HTTP status code 200, but

* a JSON response
* use appropriate response codes (`200`, `400`)